ci-operator/config/openshift/kubernetes-sigs-kueue/openshift-kubernetes-sigs-kueue-release-0.12.yaml

```diff
@@ -0,0 +1,77 @@
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN dnf install -y skopeo git && dnf clean all
+  from: src
+  to: kueue-base-src
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: vet
+  commands: make vet
+  container:
+    from: src
+- as: test
+  commands: make -f Makefile-test-ocp.mk test-ocp
+  container:
+    from: src
+- as: integration
+  commands: make -f Makefile-test-ocp.mk test-integration-ocp
+  container:
+    from: src
+- as: e2e-aws-ocp-4-19
```

Comment:
Is 4.19 the only version needed?
In 0.11 we have 4.17, 4.18 and 4.19 https://github.com/openshift/release/blob/master/ci-operator/config/openshift/kubernetes-sigs-kueue/openshift-kubernetes-sigs-kueue-release-0.11.yaml